src/Dynadot/DynadotApi.php

```diff
@@ -96,42 +110,51 @@ public function __construct($apiKey, LoggerInterface $logger = null)
                 throw new DynadotApiException('Error, received incorrect boolean value ' . $value);
             }
 
-            return $value == 'no' ? false : true;
+            return !($value == 'no');
```

Comment:
why not:
```php
return $value !== 'no';
```

Reply:
Yes, could also. This is done by phpstorm, but you are right !== is better.